cli_tools/common/utils/logging/service/log_entry.go

```diff
@@ -171,12 +171,15 @@ type OutputInfo struct {
 	FailureMessageWithoutPrivacyInfo string `json:"failure_message_without_privacy_info,omitempty"`
 }
 
-func (i *InputParams) updateParams(p *string) {
-	if p == nil {
+func (i *InputParams) updateParams(projectPointer *string) {
```

Comment:
project is enough as it's obvious it's pointer from the type

Reply:
just in order to letting project & obfuscatedProject having the same type...